demo/components/flip-reveal/sample.tsx

```
@@ -1,47 +1,40 @@
+                </ToggleGroupItem>
+                <ToggleGroupItem value="odd" className="px-4">
+                    Odd
+                </ToggleGroupItem>
+                <ToggleGroupItem value="even" className="px-4">
+                    Even
+                </ToggleGroupItem>
+            </ToggleGroup>
+            <FlipReveal className="grid grid-cols-3 gap-4" keys={[key]} showClass="flex" hideClass="hidden">
+                {Array.from({ length: 9 }).map((_, i) => (
+                    <FlipRevealItem
+                        flipKey={i % 2 == 0 ? "even" : "odd"}
```

Biome (1.9.4)

[error] Use === instead of ==. == is only allowed when comparing against `null`

== is only allowed when comparing against null

Using == may be unsafe if you are relying on type coercion
Unsafe fix: Use ===


(lint/suspicious/noDoubleEquals)